azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/collection/ResourcesInGroup.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.resources.fluentcore.collection;
+
+public interface ResourcesInGroup<T, DefinitionT> extends
+    SupportsListing<T>,
+    SupportsCreating<DefinitionT>,
+    SupportsDeleting {}
```

Comment:
so as I'm removing .get(id) from everywhere and seeing at the different implementations, I'm realizing we internally had a very different understanding of what `SupportsGetting` actually was. I had always interpreted and implemented it as getById . But it looks like Jianghao's impls were interpreting it as getByName. 
That makes it obvious we need to improve the naming here. So in the next checkin, I'm not only removing the get(id) interpretations/impls but also pointing all the impls that assumed SupportsGetting expected a name, not id, at the new interface SupportsGettingByName.
(Merging will be fun ;-)